library/src/main/java/com/wultra/android/mtokensdk/api/operation/PostApprovalScreenDeserializer.kt

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2023 Wultra s.r.o.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions
+ * and limitations under the License.
+ */
+
+package com.wultra.android.mtokensdk.api.operation
+
+import com.google.gson.JsonDeserializationContext
+import com.google.gson.JsonDeserializer
+import com.google.gson.JsonElement
+import com.wultra.android.mtokensdk.api.operation.model.*
+import com.wultra.android.mtokensdk.operation.JSONValue
+import java.lang.reflect.Type
+
+/**
+ * Gson deserializer [PostApprovalScreen]
+ *
+ * Based on "type" it returns [PostApprovalScreenReview], [PostApprovalScreenRedirect],
+ * [PostApprovalScreenGeneric] or [PostApprovalScreenUnknown]
```

Comment:
Where did the `PostApprovalScreenUnknown` come from? There seems to be no such class

Reply:
My mistake... I implemented `unknown` for PreApprovalScreen and then played a bit with unknown for PostApproval when I realized that's exactly the goal of GenericPAS (but I missed the doc)
-> removed